### PR TITLE
[docs](update) Update Doris-Streamloader docs

### PR DESCRIPTION
**Pull request by KassieZ**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by KassieZ** [comment]
run buildall

File: docs/zh-CN/docs/ecosystem/doris-streamloader.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+## 获取与安装
+
+**1.0 版本** 
+
+源代码:  https://github.com/apache/doris-streamloader
+
+| 版本    | 日期      |  平台  |  链接  |
+|---|---|---|---|
+| v1.0   |  20231110 |  x86  | https://apache-doris-releases.oss-accelerate.aliyuncs.com/apache-doris-streamloader-1.0.1-bin-x64.tar.xz|
+| v1.0   | 20231110  | aarch | https://apache-doris-releases.oss-accelerate.aliyuncs.com/apache-doris-streamloader-1.0.1-bin-arm64.tar.xz|
```
Comment (freemandealer) [inline docs/zh-CN/docs/ecosystem/doris-streamloader.md:46]: ditto

File: docs/en/docs/ecosystem/doris-streamloader.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+## Installation
+
+**Version 1.0**
+
+Source Code: https://github.com/apache/doris-streamloader/
+
+| Version | Date |  Architecture  |  Link  |
+|---|---|---|---|
+| v1.0   |  20231110 |  x86  | https://apache-doris-releases.oss-accelerate.aliyuncs.com/apache-doris-streamloader-1.0.1-bin-x64.tar.xz |
+| v1.0   | 20231110  | aarch | https://apache-doris-releases.oss-accelerate.aliyuncs.com/apache-doris-streamloader-1.0.1-bin-arm64.tar.xz |
```
Comment (freemandealer) [inline docs/en/docs/ecosystem/doris-streamloader.md:47]: Be consistent with the name in link? x86 -> x64, aarch -> arm64

File: docs/en/docs/data-operate/import/import-way/stream-load-manual.md [hunk 3]
```diff
@@ -30,6 +30,19 @@ Stream load is a synchronous way of importing. Users import local files or data
...
+
+In comparison to single-threaded load using `curl`, Doris-Streamloader is a client tool designed for loading data into Apache Doris. it reduces the ingestion latency of large datasets by its concurrent loading capabilities. It comes with the following features:
+
+- **Parallel loading**: multi-threaded load for the Stream Load method. You can set the parallelism level using the `workers` parameter.
+- **Multi-file load:** simultaneously load of multiple files and directories with one shot. It supports recursive file fetching and allows you to specify file names with wildcard characters.
+- **Path traversal support:** support path traversal when the source files are in directories
+- **Resilience and continuity:** in case of partial load failures, it can resume data loading from the point of failure.
+- **Automatic retry mechanism:** in case of loading failures, it can automatically retry a default number of times. If the loading remains unsuccessful, it will print the command for manual retry.
+
+See [Doris-Streamloader](../docs/ecosystem/doris-parallel-streamload) for detailed instructions and best practices.
```
Comment (freemandealer) [inline docs/en/docs/data-operate/import/import-way/stream-load-manual.md:43]: the url should be 'doris-streamloader'?

File: docs/zh-CN/docs/ecosystem/doris-streamloader.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+                  "dir1/basic_data1.csv"
+          ]
+  }
+  ```
+  
+- 导入失败：如果导入过程中部分数据没有导入失败了，会给出重传信息，如：
+  
+  ```Go
+  load has some error, and auto retry failed, you can retry by : 
+  ./parallel_streamload --source_file /mnt/disk1/laihui/doris/tools/tpch-tools/bin/tpch-data/lineitem.tbl.1  --url="http://127.0.0.1:8239" --header="column_separator:|?columns: l_orderkey, l_partkey, l_suppkey, l_linenumber, l_quantity, l_extendedprice, l_discount, l_tax, l_returnflag,l_linestatus, l_shipdate,l_commitdate,l_receiptdate,l_shipinstruct,l_shipmode,l_comment,temp" --db="db" --table="lineitem1" -u root -p "" --compress=false --timeout=36000 --workers=3 --batch=4096 --batch_byte=943718400 --max_byte_per_task=1073741824 --check_utf8=true --report_duration=1 --auto_retry="2,1;1,1;0,1" --auto_retry_times=0 --auto_retry_interval=60
```
Comment (freemandealer) [inline docs/zh-CN/docs/ecosystem/doris-streamloader.md:188]: parallel_streamload -> doris-streamloader

**Review by freemandealer — CHANGES_REQUESTED** [review]

**Comment by freemandealer** [comment]
@HHoflittlefish777  @dataroaring would you please take a look?

File: docs/en/docs/ecosystem/doris-streamloader.md (hunk 2 above)
Comment (freemandealer) [reply docs/en/docs/ecosystem/doris-streamloader.md:47]: Plus, the date is not updated? @KassieZ

**Review by freemandealer — APPROVED** [review]
LGTM

**Comment by freemandealer** [comment]
run buildall

**Review by luzhijing — APPROVED** [review]
LGTM

**Pull request merged**